### DISCUSSION TITLE
Upload files to a shared folder

### DISCUSSION THREAD

**Discussion by rushikharade07** [labbots/google-drive-upload#181] [Q&A] [answered]
Hello All, is there a way that I can upload files to a shared google drive folder using guppload? Thanks!

**Comment by Akianonymus** [comment, marked as answer]
Just pass the shared drive folder id with -r flag, it should work.

**Comment by rushikharade07** [comment]
How can I get the id of a new folder I created with ```-c``` flag?
For eg.
``` 
gupload -c Test
```

**Reply by Akianonymus** [reply]
When you do gupload -c Test, it prints the folder id on the terminal.

**Reply by rushikharade07** [reply]
I want to use it in a shell script. Is there a way I can get the id and save it to a variable?

**Reply by rushikharade07** [reply]
I need to further use this folder to upload some files.
For ex: 
```
Drive --> Data --> Left and Right.
```
I need to upload data to Left and RIght folders. So, I need the ID of Data to make it parent to Left and right. I creating this in a shell script for creating automatic backup of some data.

**Reply by Akianonymus** [reply]
You can get the id and save the variable, probably will need some grep and sed magic.

But if you want to upload in that test, just -c Test, and the given input will be uploaded in that folder.

**Reply by rushikharade07** [reply]
Thanks!

**Comment by rushikharade07** [comment]
I am facing issue with using gupload with cron. It seems to work fine if I try to run a script normally. But If I try to run the script in cron, it gives me Credential issues as shown below.
```
[  New account name:  ]
[ Info: Account names can only contain alphabets / numbers / dashes. ]

[ [ Error: Credentials checking failed ] ]
``` 
How do I assign default config to the script to get credentials in cron?

**Reply by Akianonymus** [reply]
> I am facing issue with using gupload with cron. It seems to work fine if I try to run a script normally. But If I try to run the script in cron, it gives me Credential issues as shown below.
> 
> ```
> [  New account name:  ]
> [ Info: Account names can only contain alphabets / numbers / dashes. ]
> 
> [ [ Error: Credentials checking failed ] ]
> ```
> 
> How do I assign default config to the script to get credentials in cron?

Use --config "path to your config" flag.

Or run cron with same username as gupload was configured initially.
